src/cli-client/internals/cli/validator.go

```diff
@@ -0,0 +1,18 @@
+package cli
+
+import (
+	"fmt"
+	"time"
+)
+
+func ValidateAndFormatDate(dateStr string) (string, error) {
```

Comment:
I may lack some context here but if this function doesn't need to be public and it is only public to test it I think it is better to use the `export_test.go` trick or to think of way of testing this through the more general function.

Reply:
Good suggestion. Added `export_test.go` to cli package.